### ISSUE TITLE
Handle errors in wit API

### ISSUE THREAD

**Issue by dwwoelfel** [wit-ai/witty-fiddle#46]
Right now, the user sees a banner about a JSON parsing error if the wit request fails. The experience would be nicer if we could show the user that we got a 500 from Wit or that the request timed out.

**Comment by stopachka** [comment]
Indeed. Some updates in App.js on the error handling could do wonders for DX
